src/sentry/static/sentry/app/components/label.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+import Tooltip from 'app/components/tooltip';
+import {Color} from 'app/utils/theme';
+
+// TODO(matej): looks like there will be more of these labels in the future.
+// Once all color combinations are designed, maybe it would make sense to
+// refactor this component to accept words like "warning" and "danger"
+// instead of manually passing backgroundColor and textColor.
+type Props = {
+  text: React.ReactNode;
+  textColor: Color;
+  backgroundColor: Color;
+  tooltip?: React.ReactNode;
+};
+
+function Label({text, textColor, backgroundColor, tooltip}: Props) {
```

Comment:
Should we call this label? Is it confusing because of form labels?

Reply:
Hm I'm not sure... I thought about "badge" or "chip" but label is still better